CIP-0067/README.md

```diff
@@ -0,0 +1,128 @@
+---
+CIP: 67
+Title: Asset Name Label Registry
+Authors: Alessandro Konrad <alessandro.konrad@live.de>, Thomas Vellekoop <thomas.vellekoop@iohk.io>
+Status: Draft
+Type: Informational
+Created: 2022-07-13
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines a standard to classify Cardano native assets by the asset name.
+
+## Motivation
+
+As more assets are minted and different standards emerge to query data for these assets, it's getting harder for 3rd parties to determine the asset type and how to proceed with it. This standard is similar to [CIP-0010](../CIP-0010), but focuses on the asset name of an asset.
+
+
+## Specification
+
+To classify assets the `asset_name` needs to be prefixed the following `4 bytes` binary encoding:
+```
+[ 0000 | 2 bytes label_num | 1 byte checksum | 0000 ]
+```
+- The leading and ending four 0s are brackets
+- `label_num` has a fixed size of 2 bytes (`Label range in decimal: [0, 65535]`). 
+If `label_num` < 2 bytes the remaining bits need to be padded with 0s.
+- `checksum` has a fixed size of 1 byte. The checksum is calculated by applying the [CRC-8](#CRC-8) algorithm on the `label_num (including the padded 0s)`. 
+
+### CRC-8
+
+- Check: `0xF4`
+- Poly: `0x07`
+- Init: `0x00`
+- RefIn: `false`
+- RefOut: `false`
+- XorOut: `0x00`
```

Comment:
I am not sure what these refer to exactly (I mean, I have only seen this notation in one particular implementation of the crc-8 algorithm in Go after searching for those specific terms). I seems that the literature mostly refer to either the lookup table and/or the polynomial representation. So it's probably sufficient to simply mention:

- Polynomial representation (normal): `0x07`
- Lookup table: ...